<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window_1</class>
 <widget class="QMainWindow" name="Main_Window_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>669</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Main_Window_1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>241</width>
      <height>154</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="floor_house">
     <item>
      <widget class="QLabel" name="label_floor">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Этажность дома</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="floor_1_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1 этаж</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="floor_1_attic_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1 этаж + мансарда</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="floor_2_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2 этажа</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="the_area_of_the_house">
     <item>
      <widget class="QLabel" name="label_area_house">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Площадь дома</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="external_area_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Внешняя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="internal_area">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Внутренняя</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>163</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="foundation">
     <item>
      <widget class="QLabel" name="label_foundation">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Фундамент</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="screw_foundation_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Винтовой</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="piled_foundation_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Свайный</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>320</y>
      <width>163</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plumbing">
     <item>
      <widget class="QLabel" name="label_plumbing">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сантехника</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="no_plumbing_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>не нужна</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="yes_plumbing_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>нужна</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>163</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="electrics">
     <item>
      <widget class="QLabel" name="label_electrics">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Электрика</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="no_electrics_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>не нужна</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="yes_electrics_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>нужна</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>167</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="underfloor_heating">
     <item>
      <widget class="QLabel" name="label_underfloor_heating">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Теплый пол</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="no_underfloor_heating_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>не нужен</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="yes_underfloor_heating_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>нужен</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="sheathings">
     <item>
      <widget class="QLabel" name="label_sheathings">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Обшивка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="gkl_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ГКЛ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="not_sh_rb">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>не нужна</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="board_walls">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>вагонка: стены</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="board_walls_ceiling">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>вагонка: стены + потолок</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="calculate_but">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Расчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_data_but">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сохранить данные</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_prices_but">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>530</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изменить цены</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>460</y>
      <width>324</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>₽</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="the_final_prices">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0                                    </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>301</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="enter_the_area_house">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_clear">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_enter_area">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>кв. м</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_enter_area">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Введите площадь 1 этажа</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>590</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
